src/components/pages/Gallery/WeeklySubmissions.js

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
+import { useOktaAuth } from '@okta/okta-react';
 import { connect } from 'react-redux';
 import Weekly from './Weekly';
 
-const WeeklySubmissions = (props) => {
+const WeeklySubmissions = props => {
```

Comment:
It looks like you may have run Prettier or an auto-formatter on this? Was there a specific reason or did it just automatically happen when you saved the file? I would not change the formatting of working code unless you've already configured the Prettier rules/settings from this repo.